src/components/StopPointsEditor/common/StopPointOrder.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconButton } from '@entur/button';
+import { BackArrowIcon } from '@entur/icons';
+import { Paragraph } from '@entur/typography';
+import React from 'react';
+
+export type StopPointOrder = {
+  order: number;
+  isFirst: boolean;
+  isLast: boolean;
+  swapStopPoints: (position1: number, position2: number) => void;
+};
+
+const StopPointOrder = ({
+  order,
+  isFirst,
+  isLast,
+  swapStopPoints,
+}: StopPointOrder) => {
+  return (
+    <div className={'order'}>
+      {!isFirst && (
+        <IconButton
+          onClick={() => swapStopPoints(order - 2, order - 1)}
+          className={'move-up'}
+          size={'small'}
+        >
+          <BackArrowIcon />
+        </IconButton>
+      )}
+      <Paragraph>{order}</Paragraph>
+      {!isLast && (
+        <IconButton
+          onClick={() => swapStopPoints(order - 1, order as number)}
```

Comment:
"as number" here doesn't seem necessary. the type of order is already "number"

Reply:
Problem was that order can be also of type undefined (because the StopPoint type is shared between all kinds of line types, and for one of them order is meant to be undefined) 😢